bentoml/_internal/exportable.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
 import os
```

Comment:
should we just do 
```
# type: ignore
```
at the beginning of this file and remove all of the inline type ignore?

Reply:
The rest of the type checking is still working fine, I don't see much reason to do this.